internal/pkg/deploy/cloudformation/app.go

```diff
@@ -496,8 +499,16 @@ func (cf CloudFormation) getLastDeployedAppConfig(appConfig *stack.AppStackConfi
 
 // DeleteApp deletes all application specific StackSet and Stack resources.
 func (cf CloudFormation) DeleteApp(appName string) error {
+	spinner := progress.NewSpinner(cf.console)
+	spinner.Start(fmt.Sprintf("Delete regional resources for application %q", appName))
```

Comment:
We want the newline char only on `.Stop` otherwise the spinner will never replace the same line 😂 everytime it spins it will print "Delete regional resources for application"

Reply:
oh lol TIL